plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/semantics/SQLCommandModelRecognizer.java

```diff
@@ -103,8 +115,23 @@ public static SQLQueryModel recognizeCommand(
             scriptContext.getVariable(varEntry.name().toUpperCase(Locale.ENGLISH));
             cmdModel.addVariable(symbolEntry, variablesResolver == null ? "?" : variablesResolver.get(varEntry.name()));
         }
+    }
 
-        return new SQLQueryModel(fakeTree, cmdModel, symbolEntries, Collections.emptyList());
+    private static void registerCommandParameter(@NotNull String text, @NotNull Set<SQLQuerySymbolEntry> symbolEntries) {
```

Comment:
(minor) Add formatting for params

Reply:
The line is not too long to use such formatting